oci/config.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+import ConfigParser
+import os
+
+gerrit_default_host = 'gerrit.ovirt.org'
+jenkins_default_host = 'jenkins.ovirt.org'
```

Comment:
Constants should be UPPER_CASE.

Also, I think we should name them:

    DEFAULT_...